src/RedisClusterOptions.php

```diff
@@ -53,6 +53,9 @@ final class RedisClusterOptions extends AdapterOptions
 
     private string $password = '';
 
+    /** @psalm-var array<non-empty-string,mixed>|null */
```

Comment:
do we know what context exists?
Can we probably have an object instead of an array?